internal/pkg/local/save.go

```diff
@@ -122,6 +122,7 @@ func (w *modelWriter) write() {
 	}
 
 	// Delete
+	w.Record.ClearRelatedPaths()
```

Comment:
Narazil som este na drobnosti, ohladom `tracked paths`. 
Po niektorych operaciach nezostal validny stav, co sa ale neprejavilo, lebo program skoncil.
Ako som (v dalsich PRs) pridal novu vrstvu `operation`, a rozdelil prikazy na mensie operacie, tak sa to prejavilo.

Reply:
1: pri ukladani je potrebne zmazat stare related paths, a ulozit nove.
Vo vacsine pripadov tam rozdiel nebude, ... ale moze nastat napr. pri `transformation blocks`, kde moze pomenovat user adresar lubovolne a my ho aj tak nacitame. Ale pri ukladani sa pouzije definovany naming.